core/io/file_access_pack.h

```diff
@@ -57,14 +57,16 @@ class PackedData {
 	friend class DirAccessPack;
 	friend class PackSource;
 
+	static PackedData *singleton;
+
 public:
 	struct PackedFile {
 		String pack;
-		uint64_t offset; //if offset is ZERO, the file was ERASED
-		uint64_t size;
-		uint8_t md5[16];
-		PackSource *src;
-		bool encrypted;
+		uint64_t offset = 0; //if offset is ZERO, the file was ERASED
+		uint64_t size = 0;
+		uint8_t md5[16]{};
```

Comment:
If we think this is too much for performance's sake, I can make this one a `// NOLINT - No init for performance.` like I've done with a few bigger arrays.